modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabaseImpl.java

```diff
@@ -173,6 +173,45 @@ public void deleteMediaPackage(String mediaPackageId, Date deletionDate) throws
     }
   }
 
+  @Override
+  public void deleteSeries(String seriesId, Date deletionDate)
+          throws SearchServiceDatabaseException, NotFoundException {
+      EntityManager em = null;
+      EntityTransaction tx = null;
+      try {
+        em = emf.createEntityManager();
+        tx = em.getTransaction();
+        tx.begin();
+
+        SearchEntity searchEntity = getSeriesSearchEntity(seriesId, em);
+        if (searchEntity == null)
+          throw new NotFoundException("No series with id=" + seriesId + " exists");
+
+        // Ensure this user is allowed to delete this series
+        String accessControlXml = searchEntity.getAccessControl();
+        if (accessControlXml != null) {
+          AccessControlList acl = AccessControlParser.parseAcl(accessControlXml);
+          User currentUser = securityService.getUser();
+          Organization currentOrg = securityService.getOrganization();
+          if (!AccessControlUtil.isAuthorized(acl, currentUser, currentOrg, WRITE.toString()))
+            throw new UnauthorizedException(currentUser + " is not authorized to delete series " + seriesId);
+
+          searchEntity.setDeletionDate(deletionDate);
+          em.merge(searchEntity);
+        }
+        tx.commit();
+      } catch (Exception e) {
+        logger.error("Could not delete series {}: {}", seriesId, e.getMessage());
```

Comment:
Proper error logging:
```suggestion
        logger.error("Could not delete series {}", seriesId, e);
```